packages/blade/src/components/Amount/_decisions/decisions.md

```diff
@@ -0,0 +1,92 @@
+# Amount <!-- omit in toc -->
+
+Amount component is a UI element that displays and formats various currency values in a specified format.
+
+<img  src="./amount-thumbnail.png"  width="380" />
+
+## Design
+
+- [Figma - Amount](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=28012%3A580578&t=3peAz8A2n2Gw4WMl-1)
+
+## API
+
+| Prop          | Type                                                                                                                                                                              | Default           | Description                                                                                                                                                                                         | Required |
+| ------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| value         | `number`                                                                                                                                                                          | `undefined`       | The value to be rendered within the component                                                                                                                                                       | ✅       |
+| intent        | `positive`, `negative`, `notice`, `information` ,`neutral`                                                                                                                        | `undefined`       | The intent of the amount to be rendered                                                                                                                                                             |
```

Comment:
i don't see `neutral` on figma. but i see `none` . can you once confirm @chaitanyadeorukhkar @Priyanshu-C what is finalised?

Reply:
Removed.